src/style.css

```diff
@@ -0,0 +1,7 @@
+.block + .block {
```

Comment:
If you want not include the first:
```suggestion
.block:not(:first-child)
```